### ISSUE TITLE
tm_fill: colorNA not effective when col >1?  

### ISSUE THREAD

**Issue by MatthieuStigler** [mtennekes/tmap#245]
Hi

I am not able to show NA values if I use more than one variable in `tm_fill()`, am I missing something?
```

library(tmap)
data(World)

World[9, c("pop_est", "pop_est_dens")] <- NA

## works
tm_shape(World) + 
  tm_fill(c("pop_est"), colorNA = "red") 

## not anymore?
tm_shape(World) + 
  tm_fill(c("pop_est", "pop_est_dens"), colorNA = "red")

```

I am using tmap 2.1.1. Thanks!!

**Comment by mtennekes** [comment]
Thanks, it's working now

**Comment by MatthieuStigler** [comment]
fixed in tmap 2.2 at least!
